web-src/src/services/AemService.js

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2023 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import { wretch } from '../helpers/NetworkHelper.js';
+
+export class AemService {
+  constructor({
+    aemHost,
+    cfEndpoint,
+    accessToken,
+  }) {
+    this.aemHost = aemHost;
+    this.cfEndpoint = cfEndpoint;
```

Comment:
- Please use `this.cfEndpoint = replaceRuntimeDomainInUrl(cfEndpoint)` by importing this function from `'../helpers/UrlHelper.js'`.
- This will fix `503 Service Unavailable` if the request takes more than 30 secs. This issue is related to the default `adobeio-static.net` domain.